src/pl.x64.json

```diff
@@ -890,7 +898,8 @@
 			"folder-name": "Papyrus",
 			"display-name": "Papyrus Script Lexer",
 			"version": "0.3.0.24",
-			"npp-compatible-versions": "[8.3.1,]",
+			"npp-compatible-versions": "[8.3,]",
+			"old-versions-compatibility": "[,0.2.2 ][,8.2.1]",
```

Comment:
For Papyrus plugin, it should be `"old-versions-compatibility": "[,0.2.3.23][,8.2.1]",`

Reply:
@blu3mania 
Thank you for you info.
However, I didn't see the version 2.3.23 in the release page:
https://github.com/blu3mania/npp-papyrus/releases

Reply:
> @blu3mania Thank you for you info. However, I didn't see the version 2.3.23 in the release page: https://github.com/blu3mania/npp-papyrus/releases

It is really this release https://github.com/blu3mania/npp-papyrus/releases/tag/v0.2.3. In the DLL, the file version info includes build #, so that's why 0.2.3 release has a binary with version 0.2.3.23, and that version was listed in the previous pl.x64.json. I am not sure how the version number comparison is handled in the code, but if it's equivalent for "0.2.3" and "0.2.3.23" then please feel free to use 0.2.3. There is no other build # for 0.2.3 release. Thanks.

Reply:
The fix is done in https://github.com/notepad-plus-plus/nppPluginList/pull/424/commits/1e13624f84205b1a6a5b2cc8d147b2bd74049934
Thank you for the explanation.